packages/react-server/core/FramebackController.js

```diff
@@ -310,6 +310,7 @@ class FramebackController extends EventEmitter {
 
 		// We've got it now, so let's set it.
 		this.setTitleFromFrame();
+		this.frame.contentWindow.focus();
```

Comment:
Nice one-line fix, @lidawang. :)

A blank line above this would make it clearer that the comment for the `setTitleFromFrame` isn't also meant to apply to the `.focus()` call.  A _comment_ for the `.focus()` call would make it even _more_ clear. ;)